### DISCUSSION TITLE
Is it possible to use the vscode jupyter extension on a website as an embedded component?

### DISCUSSION THREAD

**Discussion by dalamgir** [microsoft/vscode-jupyter#14264] [Questions and Answers]
Is it possible to use the jupyter notebook extension in a web application like a react webapp? There might be additional things that need to be implemented like jupyter services protocol etc. but was wondering if this is generally possible.

**Comment by isidorn** [comment]
No, it is not possible to run vscode extensions outside of the VS Code runtime.
